ui/packages/consul-ui/app/components/consul/exposed-path/list/index.hbs

```diff
@@ -8,7 +8,7 @@
   {{#each @items as |path|}}
     <li>
       <div class="header">
-      {{#let (concat address ':' path.Path) as |combinedAddress|}}
+      {{#let (concat @address ':' path.ListenerPort path.Path) as |combinedAddress|}}
```

Comment:
is there a separator needed between ListenerPort and Path?

Reply:
Hmm 🤔 I don't _think_ so I would think the `.Path` will need to be prefixed by `/` otherwise the routing wouldn't work
?

I'm not sure if consul enforces that, but I would guess if it doesn't and you specify `no-prefix` instead of `/no-prefix` you exposed path routing wouldn't work. Actually, if somebody does do that we'd want to show that in the UI to help them to spot the error.

I just took a look and it looks like consul doesn't enforce that, at a config level at least:

https://github.com/hashicorp/consul/blob/d32e7f08571335398c08bd6b90eff56ce9153b1a/agent/config/builder.go#L1709-L1720

@freddygv am I right in thinking if I specify:

```yaml
Address: 10.0.0.1
ListenerPort: 8080
Path: not-prefixed
# instead of
Path: /prefixed
```

The route wouldn't work? I can't see any config validation, but maybe a `/` is automatically prefixed elsewhere in the code if it doesn't exist?

Reply:
You're right. `not-prefixed` wouldn't work, paths require a leading slash and Consul doesn't prepend them if they don't exist.

Reply:
Cool, cheers Freddy! Yeah, so I'd leave it with no separator for now, that way it would look weird to somebody trying to set things up if they were to check the UI.